### PR TITLE
libcouchbase 2.7.2

### PR DESCRIPTION
**Pull request by mnunberg**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/libcouchbase.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Libcouchbase < Formula
   desc "C library for Couchbase"
   homepage "https://developer.couchbase.com/documentation/server/4.5/sdk/c/start-using-sdk.html"
-  url "https://s3.amazonaws.com/packages.couchbase.com/clients/c/libcouchbase-2.7.1.tar.gz"
-  sha256 "cd1d219e6cee21a665c61d6949f3c0cdce52526ecea878a3b73feb3ee32d6ab8"
+  url "https://s3.amazonaws.com/packages.couchbase.com/clients/c/libcouchbase-2.7.2.tar.gz"
+  sha256 "9f277619e311b53bfa4e5fa00f8fa9647ae3f9544a10e60ecc08249a3a8f252f"
   revision 1
```
Comment (fxcoudert) [inline Formula/libcouchbase.rb]: The `revision` line should go.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/libcouchbase.rb [hunk 2]
```diff
@@ -3,7 +3,6 @@ class Libcouchbase < Formula
   homepage "https://developer.couchbase.com/documentation/server/4.5/sdk/c/start-using-sdk.html"
   url "https://s3.amazonaws.com/packages.couchbase.com/clients/c/libcouchbase-2.7.2.tar.gz"
   sha256 "9f277619e311b53bfa4e5fa00f8fa9647ae3f9544a10e60ecc08249a3a8f252f"
-  revision 1
```
Comment (fxcoudert) [inline Formula/libcouchbase.rb]: Great. If you can squash those two commits together, it will be ready to merge.

**Comment by mnunberg** [comment]
I was hoping GH's UI would allow squash :/ - doing it locally.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @mnunberg for your contribution!

**Pull request closed without merging**